ui/src/app/pages/login/account-creation/account-creation-options.tsx

```diff
@@ -28,43 +24,6 @@ export const AccountCreationOptions = {
     {label: 'B.S.', value: Degree.BS},
     {label: 'B.S.N.', value: Degree.BSN},
   ],
-  // BEGIN roles and affiliations for old (pre-verification) institutional affiliation
```

Comment:
components referencing these were removed on a previous sweep

Reply:
I'd say check Rainforest scripts, but that's already retired, right?

Reply:
This can be demonstrated simply by compilation

Reply:
Right, but if we did have references to these labels in Rainforest scripts or help text, that wouldn't be caught by the compiler. Just double-checking.

Reply:
Nothing visible to Rainforest is changing in this PR